<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FieldQuickReference</class>
 <widget class="QWidget" name="FieldQuickReference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Field Quick Reference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,10">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDescriptor">
       <property name="text">
        <string>Descriptor:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxDescriptor"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelField">
       <property name="text">
        <string>Field:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxField"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
